src/torrent/utils/thread_base.cc

```diff
@@ -60,7 +61,7 @@ thread_base::stop_thread_wait() {
   release_global_lock();
 
   while (!is_inactive()) {
-    usleep(1000);
+    std::this_thread::sleep_for(std::chrono::milliseconds(1));
```

Comment:
Looks like average thread join. Sleeping for join is anti-pattern in userspace.

Reply:
correct